python/sources/web_api_gateway/README.md

```diff
@@ -4,10 +4,14 @@
 
 ## How to run
 
-Create a [Quix](https://portal.platform.quix.ai/self-sign-up?xlink=github) account or log-in and visit the Samples to use this project.
-
 Clicking `Edit code` on the Sample, forks the project to your own Git repo so you can customize it before deploying.
 
+Once you have deployed the Sample within your project there are three things to bear in mind:
+
+1. Your data JSON must contain a "sessionId" key 
+2. Keys and values must be Strings or Bytes.
+3. The Flask gateway endpoint must end with `/data/`. E.g `https://gateway-example-develop.deployments.quix.io/data/`
+
```

Comment:
you could add an example of how to post to it using curl

Reply:
I've added an example. Do I need to explain how to consume the message? I.e through a topic? I know how to explain this via Quix Portal but I don't know how to explain this via the CLI. Does this code automatically generate a topic when ran locally? Or can this application only be ran on QP?

Reply:
I've added an example. Do I need to explain how to consume the message? I.e through a topic? I know how to explain this via Quix Portal but I don't know how to explain this via the CLI. Does this code automatically generate a topic when ran locally? Or can this application only be ran on QP?